cli/operator/node.go

```diff
@@ -97,7 +99,12 @@ var StartNodeCmd = &cobra.Command{
 		}
 		nodeStorage, operatorData := setupOperatorStorage(logger, db)
 
-		keyManager, err := ekm.NewETHKeyManagerSigner(logger, db, networkConfig, cfg.SSVOptions.ValidatorOptions.BuilderProposals)
+		operatorKey, _, _ := nodeStorage.GetPrivateKey()
```

Comment:
I would say lets check for error but after `setupOperatorStorage` we should have it for sure.

Reply:
this is why I removed it,
in that step the private key must be set or the node won't run